src/rust/bitbox02/src/sd.rs

```diff
@@ -30,6 +30,20 @@ pub fn sdcard_inserted() -> bool {
     data.sdcard_inserted.unwrap()
 }
 
+#[cfg(not(feature = "testing"))]
+pub fn sdcard_vfat_formatted() -> bool {
+    unsafe { bitbox02_sys::sd_card_vfat_formatted() }
+}
+
+#[cfg(feature = "testing")]
+pub fn sdcard_vfat_formatted() -> bool {
+    true
+}
+
+pub fn sd_format() -> bool {
```

Comment:
`-> Result<(), ()>` is more idiotmatic Rust for error handling. You can see other examples for converting C bools to Results in the bitbox02/src files.

Reply:
Okay, converting that to Result object